PIMbench/cpp-knn/baselines/CPU/knn.cpp

```diff
@@ -0,0 +1,280 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <unistd.h>
+#include <getopt.h>
+#include <assert.h>
+#include <stdint.h>
+#include <iostream>
+#include <fstream>
+#include <sstream>
+#include <cmath>
+#include <unordered_map>
+#include <chrono>
+#include <vector>
+#include <string>
+#include <queue>
+#include <iomanip>
+using namespace std;
+
+#include <omp.h>
+#include "../../../utilBaselines.h"
+#include <cfloat>
+
+vector<vector<int>> dataPoints;
+vector<vector<int>> testPoints;
+
+// Params ---------------------------------------------------------------------
+typedef struct Params
+{
+    int numTestPoints;
+    int numDataPoints;
```

Comment:
```suggestion
    uint64_t numTestPoints;
    uint64_t numDataPoints;
```
this is to avoid overflow in case of very large number. Could you please update all the usage of this accordingly?

Reply:
@kmd2zjw I think you have missed this. Could you please check?